doc/api/async_hooks.md

```diff
@@ -29,7 +29,7 @@ Following is a simple overview of the public API.
 const async_hooks = require('async_hooks');
 
 // Return the ID of the current execution context.
-const cid = async_hooks.currentId();
+const cid = async_hooks.currentAsyncId();
```

Comment:
Great. @AndreasMadsen seems like `executionAsyncId()` and `triggerAsyncId()` are the names to use. Thanks for taking care of this.